### PR TITLE
Fix docs generation locally on Windows.

### PR DESCRIPTION
**Pull request by Sergeus**
## Proposed change
The "Manual Environment" described in the [docs for setting up a local workflow](https://developers.home-assistant.io/docs/documenting/) for working on the HA docs fails on Windows (using mingw) at the moment.

These changes got it working for me, but I'm no Ruby expert, so feel free to tell me to go pound sand with this PR.

Two upgrades were required:

* Sassc from 2.1.0 to 2.4.0. This resolved a C++ compiler error with std::unordered_set when trying to install the Sass gem (happened while running the initial `bundle` in the installation instruction steps).
* Wdm from 0.1.0 to 0.2.0. This resolved an issue with implicit function declarations that appears to be a change within Ruby itself (also happened during initial `bundle`). Version 0.2.0 of wdm was [published to address this issue and support newer Ruby versions last month](https://github.com/Maher4Ever/wdm/pull/29). I dropped back to run with Ruby 3.1.6, after I found [this PR](https://github.com/home-assistant/home-assistant.io/pull/30538) that mentioned 3.2+ is not supported yet since the removal of `exists?` breaks Compass. (Since wdm is Windows only, I suspect folks haven't been running locally on Windows with a manual environment since that upgrade.)

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Chores**
	- Updated the `sassc` gem to version `2.4.0` for improved performance and compatibility.
	- Updated the `wdm` gem to version `0.2.0` to enhance functionality for Windows users.

### CODE REVIEW

**Comment by Sergeus** [comment]
I'm a bit confused by the bot action, I don't *think* this should be targeting the `next` branch, because all of the other PRs I can find that modify the gemfile were merged into the `current` branch.

File: Gemfile [hunk 1]
```diff
@@ -33,4 +32,4 @@ install_if -> { RUBY_PLATFORM =~ %r!mingw|mswin|java! } do
 end
 
 # Performance-booster for watching directories on Windows
-gem "wdm", "~> 0.1.0" if Gem.win_platform?
+gem "wdm", "~> 0.2.0" if Gem.win_platform?
```
[the remark this answers is not in the document]
Comment (Sergeus) [reply Gemfile:35]: I used 3.1.6 locally, but 3.1.4 is also fine. Using 3.+ at all with wdm requires 0.2.0, which is what I was talking about in the body of the PR. (I know this is bot feedback, but I'm recording this here for any readers who see it and wonder)

File: Gemfile [hunk 2]
```diff
@@ -8,8 +8,7 @@ group :development do
   gem 'compass', '1.0.3'
   gem 'sass-globbing', '1.1.5'
   gem 'stringex', '2.8.6'
-  # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
-  gem 'sassc', '2.1.0'
+  gem 'sassc', '2.4.0'
```
Comment (frenck) [inline Gemfile:11]: We cannot change this, as per the comment on the existing code.

See: https://github.com/sass/sassc-ruby/issues/189

If this is problematic for Windows, I think it is better to start documenting you cannot use this on Windows.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (Sergeus) [reply Gemfile:11]: Oh, I completely misread the purpose of that comment! I'll try to sort this out with the older version - see if there's some other way to make that gem compatible to be installed at 2.1.0

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

**Pull request closed without merging**